src/Directory.Build.targets

```diff
@@ -138,6 +138,52 @@
 
   <Import Project="$(BUILD_STAGINGDIRECTORY)\MicroBuild\Plugins\MicroBuild.Plugins.IBCMerge.*\**\build\MicroBuild.Plugins.*.targets" Condition="'$(BUILD_STAGINGDIRECTORY)' != '' and $(TargetFramework.StartsWith('net4')) and '$(MicroBuild_EnablePGO)' != 'false'" />
 
+  <PropertyGroup>
+    <UseFrozenMaintenancePackageVersions Condition="'$(UseFrozenMaintenancePackageVersions)' == '' AND '$(IsUnitTestProject)' != 'true' AND $([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net472')) AND '$(OutputType)' != 'exe'">true</UseFrozenMaintenancePackageVersions>
```

Comment:
Why net472 specifically? Just wondering since 4.6.2 is the [minimum supported](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework).

I suppose it's because the minimum supported version by Microsoft.IO.Redist is 4.7.2?

Reply:
Yeah this is repo-specific; I mean "is .NET Framework but not our net35 stuff". I'll comment in a follow-up.